app/models/ptable.rb

```diff
@@ -33,6 +33,8 @@ class Ptable < Template
 
   alias_attribute :layout, :template
 
+  attr_exportable :os_family
```

Comment:
I decided to add :snippet to base class because of following reasons
* consistency with ptables
* job templates (rex plugin) consitency
* foreman_templates can simplify [the condition](https://github.com/theforeman/foreman_templates/blob/master/app/models/concerns/foreman_templates/provisioning_template_import.rb#L7)

one downside is that since it's boolean, it will be always present, mostly as false. We could probably move the scope search definition to base class too, but that's a different story.